test/integration_tests/test_insert_executor.py

```diff
@@ -60,7 +59,8 @@ def test_should_insert_row_in_table(self):
         insertExec.exec()
 
         # test if we have a added the in our storage
-        df = load_dataframe(url)
+        df = None  # will fix later
+        # df = load_dataframe(url)
```

Comment:
I think this is still a work in progress. So we should drop this for now and maybe add with a later PR.

Reply:
Fixed by 6de611c